app/code/Magento/Customer/Ui/Component/Listing/Address/Column/Actions.php

```diff
@@ -6,6 +6,9 @@
  */
 namespace Magento\Customer\Ui\Component\Listing\Address\Column;
 
+use Magento\Customer\Api\AddressRepositoryInterface;
```

Comment:
Hi @atwixfirster. Thanks for collaboration. Looks like this code not related to this PR.

Reply:
Very good point, @VladimirZaets 

Yes, I 've mistakenly committed wrong files. 

Thank you!